datalad_next/create_sibling_webdav.py

```diff
@@ -470,32 +543,29 @@ def _create_git_sibling(
         # e.g., it is not uncommon for credentials to be named after URLs
         remote_url += f'&dlacredential={urlquote(credential_name)}'
 
-    yield from ds.siblings(
-        # TODO set vs add, consider `existing`
-        action="add",
+    return ds.siblings(
+        action="add" if not reconfigure else "configure",
         name=name,
         url=remote_url,
-        # TODO probably needed when reconfiguring, but needs credential patch
-        fetch=False,
-        publish_depends=dependency,
-        return_type='generator',
+        publish_depends=publish_depends,
+        return_type='item-or-list',
         result_renderer='disabled')
 
 
-def _create_storage_sibling(ds, url, name, existing, credential, export):
+def _create_storage_sibling(ds, url, name, credential, export, reconfigure):
```

Comment:
Same comment as above, passing `existing` is more future proof and hardly more complex.

Reply:
done